swagger/event_variables.py

```diff
@@ -3,839 +3,851 @@
 
 class Responce:
     EventResponse = openapi.Response(
-            'Event created successfully',
-            schema=openapi.Schema(
-                type=openapi.TYPE_OBJECT,
-                required=[
-                    'id', 'name', 'competition_type', 'date_from', 'date_to',
-                    'place', 'place_region', 'organizer', 'distances'
-                ],
-                properties={
-                    'id': openapi.Schema(
-                        type=openapi.TYPE_INTEGER,
-                        description='Event ID'
-                    ),
-                    'name': openapi.Schema(
-                        type=openapi.TYPE_STRING,
-                        description='Event name'
+        'Event created successfully',
+        schema=openapi.Schema(
+            type=openapi.TYPE_OBJECT,
+            required=[
+                'id', 'name', 'competition_type', 'date_from', 'date_to',
+                'place', 'place_region', 'co_organizer', 'organizer', 'distances'
```

Comment:
Delete "co_organizer". This field will not necessarily be returned, since, for example, when the event is created, you do not specify any values ​​for the co_organize field

Reply:
ок